_commands/basics/jobs.md

```diff
@@ -3,22 +3,99 @@
 
 jobs
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+
+`jobs` is used to list the jobs running in the background of the current shell session.
+
+The general format is
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ jobs [options] jobID
```

Comment:
the square brackets seem to imply that the options are optional, but because there aren't brackets around the jobID, it implies that the `jobID` argument is required. most of your example commands don't include a `jobID`, so is it actually optional? and if so, it should also include another set of square brackets `[]`.